docs/developer_guide/SimpleSensorSimulator.md

```diff
@@ -2,25 +2,98 @@
+| invisible grid | 50    | a grid that is out of range or occlusion of simulated lidar sensor  |
+
+
+### Configuration
+| name              | type   | unit   | description                                                                                                                         |
+|-------------------|--------|--------|-------------------------------------------------------------------------------------------------------------------------------------|
+| entity            | string | -      | Name of the entity which you want to attach detection sensor                                                                        |
+| update_duration   | double | second | Update duration of the detection sensor                                                                                             |
+| resolution        | double | meter  | Resolution of the occupancy grid                                                                                                    |
+| width             | uint32 | pixel  | Width of the occupancy grid                                                                                                         |
+| height            | uint32 | pixel  | Height of the occupancy grid                                                                                                        |
+| architecture_type | string | -      | Autoware architecture type                                                                                                          |
+| range             | double | meter  | Sensor detection range                                                                                                              |
+| filter_by_range   | bool   | -      | True: simulator publish detection result of entities in range<br/>False : simulator publish detection result only lidar ray was hit |
+
+
+## Object Detection Results Simulation
+With this simulation, you can get object detection results without processing images by heavy object detection algorithms.  
+This also enables you to reduce computational resources when you want to test Autoware's planners or controllers.
+
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachDetectionSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_detection_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5564 |  
+
+
+### Configuration
+
```

Comment:
Please remove this table.
Configuration should be auto-generated from protobuf-gen-c-doc